mava/configs/env/mabrax.yaml

```diff
@@ -13,6 +13,10 @@ scenario:
 # This metric is returned at the end of an experiment and can be used for hyperparameter tuning.
 eval_metric: episode_return
 
+# Whether the environment observations encode implicit agent IDs. If True, the AgentID wrapper is not used.
+# This should not be changed.
+implicit_agent_id: False
```

Comment:
This should be True since we're using the `homogenisation_method: max` that adds the IDs